tsconfig.json

```diff
@@ -26,7 +26,6 @@
     "node_modules",
     "build",
     "demo",
-    "tst",
```

Comment:
I think in the past Harshit tried to run linting on `tst` folder and we ran into various issues. Any reason we are including it now?

Reply:
I tried linting in `tst` folder and it is working fine. Actually, this removal on line 29 is really sort of a bug because up on line 24 we are including the `tst` folder: https://github.com/aws/amazon-chime-sdk-component-library-react/blob/main/tsconfig.json#L24. The config is basically adding and removing the same folder. With the `tst` folder in the exclude I have noticed that the linter is throwing following parser errors:
```
/Users/kunalnan/Documents/AWS/amazon-chime-sdk-component-library-react/tst/providers/MeetingProvider/index.test.tsx
  0:0  error  Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: tst/providers/MeetingProvider/index.test.tsx.
The file must be included in at least one of the projects provided
```

Reply:
I have added the `tst` exclusion back in as removing it seems to be causing issues with Storybook `npm run start`. I tried to adjust the exclusion but a lot of care is need to get this right. So, I have left a TODO comment for now.